sdk/vppagent/commit.go

```diff
@@ -23,99 +23,81 @@ const (
 	createConnectionSleep   = 100 * time.Millisecond
 )
 
-// Flush is a VPP Agent Flush composite
-type Flush struct {
-	endpoint.BaseCompositeEndpoint
+// Commit is a VPP Agent Commit composite
+type Commit struct {
 	Endpoint string
 }
 
 // Request implements the request handler
-func (f *Flush) Request(ctx context.Context, request *networkservice.NetworkServiceRequest) (*connection.Connection, error) {
-	if f.GetNext() == nil {
-		err := fmt.Errorf("composite requires that there is Next set")
-		return nil, err
-	}
-
-	incomingConnection, err := f.GetNext().Request(ctx, request)
-	if err != nil {
-		return nil, err
+// Provides/Consumes from ctx context.Context:
+//     VppAgentConfig
+//	   Next
+func (f *Commit) Request(ctx context.Context, request *networkservice.NetworkServiceRequest) (*connection.Connection, error) {
```

Comment:
Please rename the receiver names to `c` also

Reply:
Good catch :)  Will do :)